packages/babel-core/src/transformation/file/options/option-manager.js

```diff
@@ -179,9 +179,17 @@ export default class OptionManager {
       // check for an unknown option
       if (!option && this.log) {
         let pluginOptsInfo = "Check out http://babeljs.io/docs/usage/options/ for more info";
+        let presetConfigInfo = "For correct configuration, please see http://babeljs.io/docs/plugins/#pluginpresets-options.";
 
         if (removed[key]) {
           this.log.error(`Using removed Babel 5 option: ${alias}.${key} - ${removed[key].message}`, ReferenceError);
+        } else if (opts.presets && Array.isArray(opts.presets)) {
+          for (let i = 0; i < opts.presets.length; i++) {
+            let preset = opts.presets[i];
+            if (typeof preset === "object" && preset !== null && !Array.isArray(preset)) {
+              this.log.error(`Found configuration options ${JSON.stringify(opts)} without corresponding preset. ${presetConfigInfo}`);
```

Comment:
Seemed like a lot to log, which is why I put the link in. Can add that in if you think it's better though!